docs/src/examples/components/Toolbar/Content/index.tsx

```diff
@@ -25,6 +25,11 @@ const Content = () => (
       description="Toolbar items can be grouped into radio group. Up/Down arrow keys can be used to cycle between radio items. Only one of the radio items can be selected at a time, should be implemented additionally."
       examplePath="components/Toolbar/Content/ToolbarExampleRadioGroup"
     />
+    <ComponentExample
+      title="Toolbar can contain custom content"
+      description="Toolbar item can contain custom content."
```

Comment:
I would add here or in the accessibility section of the whole Toolbar page, that when custom item used, accessibility and styles should be covered manually

Reply:
Section was added 👍